### PR TITLE
feat: Add Spanish translations

### PR DESCRIPTION
**Pull request by manuelmhtr**
## 🎯 Goal

Add translations file for Spanish.

## 🛠 Implementation details

* Added Spanish translations
* Added missing references for existing translations (Hebrew, Japanese, Korean)
* Ordered translation references alphabetically by language id.

## 🎨 UI Changes

<details>
<summary>iOS</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

<details>
<summary>Android</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

## 🧪 Testing

By configuring the app with the `es` language.

## ☑️ Checklist

- [X] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [X] PR targets the `develop` branch
- [X] Documentation is updated
- [X] New code is tested in main example apps, including all possible scenarios
  - [X] SampleApp iOS and Android
  - [X] Expo iOS and Android

### CODE REVIEW

File: package/src/i18n/es.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  "Video": "Video",
+  "You": "Tú",
+  "You can't send messages in this channel": "No puedes enviar mensajes en este canal",
+  "{{ firstUser }} and {{ nonSelfUserLength }} more are typing": "{{ firstUser }} y {{ nonSelfUserLength }} más están escribiendo",
+  "{{ index }} of {{ photoLength }}": "{{ index }} de {{ photoLength }}",
+  "{{ replyCount }} Replies": "{{ replyCount }} Respuestas",
+  "{{ replyCount }} Thread Replies": "{{ replyCount }} respuestas de hilo",
+  "{{ user }} is typing": "{{ user }} está escribiendo",
+  "🏙 Attachment...": "🏙 Adjunto..."
+}
```
Comment (khushal87) [inline package/src/i18n/es.json:70]: We just added a new translations for `Maximum file size upload limit reached. Please upload a file below {{MAX_FILE_SIZE_TO_UPLOAD_IN_MB}} MB.` and there is one coming for `Select More Photos` in https://github.com/GetStream/stream-chat-react-native/pull/2291/files.

Can you please add them? Thanks 😄

**Review by khushal87 — CHANGES_REQUESTED** [review]

File: package/src/i18n/es.json (hunk 1 above)
Comment (manuelmhtr) [reply package/src/i18n/es.json:72]: No problem. Done 👍

**Comment by khushal87** [comment]
Can you run `yarn lint-fix` and push the changes? Seems like linting checks are failing for the PR. @manuelmhtr

**Comment by manuelmhtr** [comment]
> Can you run `yarn lint-fix` and push the changes? Seems like linting checks are failing for the PR. @manuelmhtr

Sorry about that. It's fixed now.

**Review by khushal87 — APPROVED** [review]

**Pull request merged**